lib/models/addon.js

```diff
@@ -8,7 +8,9 @@ var existsSync   = require('exists-sync');
 var path         = require('path');
 var assign       = require('ember-cli-lodash-subset').assign;
 var SilentError  = require('silent-error');
-var logger       = require('heimdalljs-logger')('ember-cli:addon');
+var heimdallLogger = require('heimdalljs-logger');
+var logger = heimdallLogger('ember-cli:addon');
```

Comment:
It is definitely used (this PR just moved things around a tad to avoid calling `require('heimdalljs-logger')` multiple times) in `initializeAddons`.